assets/prototype/domain/shared/dateRangeInput/DateRangePicker.js

```diff
@@ -1,21 +1,21 @@
+import moment from 'moment';
```

Comment:
Ideally, we should have our own `format` function somewhere in datetime utils and in that file we can swap in the future `moment` with whatever we want.

Reply:
yay that gets tricky because the different libs use different formats, but I guess that's the point of adapters eh?

Reply:
yeah and we don't want to paint yourself in the corner with a lib.